.github/workflows/wheels.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         with:
           python-version: "3.10"
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel
+        run: python -m pip install "cibuildwheel<2.15.0"
```

Comment:
or alternatively you could let cibuildwheel unpinned and add python 3.12 to CIBW_SKIP variable, with a TODO comment about fixing the deprecation errors later on:

https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip

Reply:
Now we have two alternatives. What would you prefer?

Reply:
i prefer we keep cibuildwheel to latest (unpinned) version and explicitly (temporarily) skip python 3.12 until we fix #18

Reply:
Please take a look again. Is the formulation good enough?

Reply:
Oh nevermind you force pushed that, yes LGTm